target/mips/translate.c

```diff
@@ -10759,7 +10764,7 @@ static void gen_mftr(CPUMIPSState *env, DisasContext *ctx, int rt, int rd,
         }
     }
     trace_mips_translate_tr("mftr", rt, u, sel, h);
-    gen_store_gpr(t0, rd);
+    gen_store_gpr(ctx, t0, rd);
```

Comment:
I think for RISCV I avoided these diffs to upstream using this ugly hack:
`#define gen_store_gpr(val, rd) _get_store_gpr(ctx, val, rd)`
Not ideal but will reduce compilation failures after upstream merges.

Reply:
I noticed that it does it. I was conflicted on whether to do the same and I avoided doing it for now on the basis that it is an ugly hack and that there are various other logging additions that may break merges in a similar way. I'll consider removing this diff if it makes it hard to merge upstream merges.